ios/Podfile

```diff
@@ -37,6 +37,14 @@ target 'Runner' do
   end
 end
 
+target 'NotificationService' do
+  use_frameworks!
+
+  pod 'BreezSDKLiquid', '= 0.2.2-dev7'
```

Comment:
Do we need to manually update this version on each release?
How does this effect working locally with a locally built library & plugin?

Reply:
> Do we need to manually update this version on each release? How does this effect working locally with a locally built library & plugin?

This is just for testing as it's only available in pre-release. Can be changed when released:
```
  pod 'BreezSDKLiquid'
```